File: IMC/IMC/Models/Imc.cs
namespace IMC.Models;

public class Imc {

    public string ImcId { get; set; } = Guid.NewGuid().ToString();
    public Aluno? Aluno { get; set; }
    public string? AlunoId { get; set; }
    public double Altura { get; set; }
    public double Peso { get; set; }
    public double Valor { get; set; }
    public string Classificacao { get; set; }
    public DateTime DataCriacao { get; set; } = DateTime.Now;
}

File: front/src/models/imc.ts
import { Aluno } from "./aluno";

export interface Tarefa {
  imcId?: string;
  altura: number;
  peso: number;
  valor?: number;
  classificacao?: string;
  alunoId?: string;
  aluno?: Aluno;
  dataCriacao?: string;
}
